Makefile

```diff
@@ -463,6 +470,14 @@ release: ## Re-generate generated code and run tests
 	$(MAKE) --no-print-directory go-generate
 	$(MAKE) --no-print-directory test
 
+build: ## Build all packages and all tests (ensures everything compiles)
+	$Q echo 'Building all packages...'
+	$Q go build ./...
+	$Q # "tests" by building and then running `true`, and hides test-success output
+	$Q echo 'Building all tests (~5x slower)...'
+	$Q # intentionally not -race due to !race build tags
+	$Q go test -exec /usr/bin/true ./... >/dev/null
```

Comment:
I like this option `-exec /usr/bin/true` to just verify test build

Reply:
I've found it on a few go-official channels (groups and github), it's... very effective but I do wish it were just `go build -tests ./...` or something.  Kinda hard to remember.